main.go

```diff
@@ -115,34 +126,38 @@ func main() {
 	// Initialize event recorder.
 	record.InitFromRecorder(mgr.GetEventRecorderFor("gcp-controller"))
 
+	// Setup the context that's going to be used in controllers and for the manager.
+	ctx := ctrl.SetupSignalHandler()
+
 	if webhookPort == 0 {
```

Comment:
I believe this conditional should be removed, since the controllers and webhooks should be sharing the same deployment now.